app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java

```diff
@@ -1577,6 +1582,12 @@ private void checkNothingFavourites() {
             starImageView.setColorFilter(ContextCompat.getColor(this, R.color.black), PorterDuff.Mode.SRC_ATOP);
     }
 
+    private void checkNoSearchResults(String result){
+        textView.setText(getString(R.string.null_search_result) + " " + '"' + result + '"' );
```

Comment:
In the string there's already space at the end, do we need two spaces?

Reply:
@abishekvashok Removed the space at the end of the string declaration in strings.xml file. Please review.